docs/ecosystem.rst

```diff
@@ -20,7 +20,9 @@ Community Tutorials
 
 * `Build data-driven applications with Workers and PostgreSQL <https://developers.cloudflare.com/workers/tutorials/postgres/>`_ - A tutorial on how to integrate with PostgREST and PostgreSQL using Cloudflare Workers.
 
-.. * `A poor man's API <https://blog.frankel.ch/poor-man-api>`_ - Shows how to integrate PostgREST with Apache APISIX as an alternative to Nginx.
+* `A poor man's API <https://blog.frankel.ch/poor-man-api>`_ - Shows how to integrate PostgREST with Apache APISIX as an alternative to Nginx.
+
+* `Accessing a PostgreSQL database in Godot 4 via PostgREST <https://peterkingsbury.com/2022/08/16/godot-postgresql-postgrest/>`_
```

Comment:
Let's try with plain HTTP

```suggestion
* `Accessing a PostgreSQL database in Godot 4 via PostgREST <http://peterkingsbury.com/2022/08/16/godot-postgresql-postgrest/>`_
```

Reply:
Unfortunately, it still redirects to the http page.